src/Configurator/AddLinesConfigurator.php

```diff
@@ -164,10 +164,14 @@ public function executeUnconfigure(Recipe $recipe, $config): void
         }
     }
 
-    private function getPatchedContents(string $file, string $value, string $position, ?string $target, bool $warnIfMissing): string
+    private function getPatchedContents(string $file, string|array $value, string $position, ?string $target, bool $warnIfMissing): string
     {
         $fileContents = $this->readFile($file);
 
+        if (is_array($value)) {
+            $value = implode(PHP_EOL, $value);
```

Comment:
Please use `"\n"` instead of `PHP_EOL`. We don't want to generate a mix of LF and CRLF on Windows (and we generate code with LF)

Reply:
Thanks for your feedback!